components/article-footer/server.js

```diff
@@ -36,11 +36,10 @@ export class ArticleFooter extends ServerComponent {
  */
 function Contribute(context) {
   return html`<a
-    class="article-footer__contribute external"
-    href="https://github.com/mdn/content/blob/main/CONTRIBUTING.md"
-    title=${context.l10n`This will take you to our contribution guidelines on GitHub.`}
+    class="article-footer__contribute"
+    href=${`/{context.locale}/docs/MDN/Community/Getting_started`}
+    title=${context.l10n`This will take you to our contribution documentation.`}
```

Comment:
Nit: I think this title no longer makes sense, as it just duplicates the text.

```suggestion
```

Reply:
I'm +1 on removing, does this mostly make sense for external links?

Reply:
Actually, I realize that this link opens in a new tab. If we keep that, then the title should have a suffix "(opens in new tab)".

In general, I would say a `title` makes sense only where the link text is ambiguous.

Reply:
Thanks, both. I think this is ready, now!